### PR TITLE
EOS-15932 No verbose mode present in hctl-shutdown

### PR DESCRIPTION
**Pull request by SwapnilGaonkar7**
Used argparse to handle argument and modified code to enable veebose mode when -v is passed

### CODE REVIEW

File: utils/hare-shutdown [hunk 1]
```diff
@@ -109,23 +110,30 @@ def node_shutdown():
...
+                        help="Node shutdown",
+                        action="store_true")
+
+    args = parser.parse_args()
+    if args.verbose:
+        logging.basicConfig(level=logging.DEBUG, format='%(message)s')
+    else:
+        logging.basicConfig(level=logging.INFO, format='%(message)s')
+
+    skip_consul_stop = True if args.skip_consul_stop else False
```
Comment (knekrasov) [inline utils/hare-shutdown:131]:
```suggestion
    skip_consul_stop = bool(args.skip_consul_stop)
```

Comment (SwapnilGaonkar7) [reply utils/hare-shutdown]: Done

**Review by knekrasov — APPROVED** [review]

File: utils/hare-shutdown [hunk 2]
```diff
@@ -109,23 +108,30 @@ def node_shutdown():
...
+    parser.add_argument("--skip-consul-stop",
+                        help="Skip consul service stop",
+                        action="store_true")
+    parser.add_argument("--node",
+                        help="Node shutdown",
+                        action="store_true")
+
+    args = parser.parse_args()
+    if args.verbose:
+        logging.basicConfig(level=logging.DEBUG, format='%(message)s')
```
Comment (mssawant) [inline utils/hare-shutdown:125]: Do we want it the other way around, i.e. Info in case of verbose and Debug otherwise?

Comment (knekrasov) [reply utils/hare-shutdown:125]: Verbose is usually understood as 'too wordy' or  'reporting (too) much'. 

In case DEBUG level is set, log messages of both INFO and DEBUG will be seen. If INFO level is set, only INFO messages will be seen. So per my understanding, the code makes sense.

Comment (SwapnilGaonkar7) [reply utils/hare-shutdown:125]: @mssawant Agree with Konstantin. Verbose should produce more detailed logs.

Comment (mssawant) [reply utils/hare-shutdown:125]: >In case DEBUG level is set, log messages of both INFO and DEBUG will be seen. If INFO level is set, only INFO messages will be seen

@knekrasov, @SwapnilGaonkar7, is this a universal assumption or just with respect to Python? I think in any case we should explicitly state this assumption in form of a comment here itself.

**Review by mssawant — APPROVED** [review]
Minor documentation comment.

File: utils/hare-shutdown (hunk 2 above)
Comment (SwapnilGaonkar7) [reply utils/hare-shutdown:133]: @mssawant  Done

**Review by mssawant — APPROVED** [review]
